tests/test_mock_controller.py

```diff
@@ -65,90 +68,811 @@ async def test_missing_command_parameter(self):
         self.data = await self.read()
         sau.assertReply("ERROR", self.data, CODE=3)
 
-    async def test_status(self):
-        await self.write("status:\n")
-        self.data = await self.read()
-        sau.assertReply("AMCS", self.data, status="Stopped", positionActual=0)
-        sau.assertReply("ApCS", self.data, status="Stopped", positionActual=0)
-        sau.assertTBD("LCS", self.data)
-        sau.assertTBD("LWCS", self.data)
-        sau.assertTBD("ThCS", self.data)
-        sau.assertTBD("MonCS", self.data)
-
     async def test_moveAz(self):
         await self.write("moveAz:\n azimuth: 10\n")
         self.data = await self.read()
         sau.assertReply("OK", self.data, Timeout=20)
+        await asyncio.sleep(1)
```

Comment:
I'll add comments. Also I am open to suggestions on how to speed up the test. I tried using an asynctest.ClockedTestCase instead of asynctest.TestCase but that didn't work. I prefer not to speed up the mock devices since they will be used in non-unit test cases as well.

Reply:
When I speed up mock drives I typically do it only for the unit test, by changing the motion parameters. Depending on the API that may be a reasonable or unreasonable thing to do. In any case it was just a thought; it's not worth a lot of work to shave a few seconds off a test.